README.md

```diff
@@ -52,10 +55,41 @@ const pool = new Pool({
   database: 'db-name',
   max: 5
 });
-const result = await pool.query('SELECT NOW()');
+const {rows} = await pool.query('SELECT NOW()');
+console.table(rows) // prints returned time value from server
+
+await pool.end();
+connector.close();
+```
+
+### Using with MySQL
+
+The connector package is meant to be used alongside a database driver, in the
```

Comment:
This is identical to the paragraph above. Should we just move consolidate both paragraphs just below `##Usage` and then say here (and under Postgres) something like, "The Connector supports mysql2 and can be configure like so:"?

Reply:
sure! let me give it a try!